src/models/use_fans_heatwave.js

```diff
@@ -0,0 +1,253 @@
+        tdb: tdb,
+        tr: tr,
+        v: v,
+        area: body_surface_area,
+        pressure: p_atm,
+      },
+      "IP",
+    );
+    tdb = unit_convert.tdb;
+    tr = unit_convert.tr;
+    v = unit_convert.v;
+    body_surface_area = unit_convert.area;
+    p_atm = unit_convert.pressure;
+  }
+
+  let output = two_nodes(
+    tdb,
+    tr,
+    v,
+    rh,
+    met,
+    clo,
+    wme,
+    body_surface_area,
+    p_atm,
+    body_position,
+    max_skin_blood_flow,
+    { round: false, max_sweating: joint_kwargs.max_sweating },
+  );
+
+  let heatwave_output = cal_heatwave(output, joint_kwargs, max_skin_blood_flow);
+
+  const key_to_remove = ["set", "et", "pmv_gagge", "pmv_set", "disc", "t_sens"];
+
+  const joint_output = {};
+  for (const key in output) {
+    if (!key_to_remove.includes(key)) {
+      joint_output[key] = output[key];
+    }
+  }
+  Object.assign(joint_output, heatwave_output);
+
+  if (joint_kwargs.limit_inputs) {
+    const {
+      tdb: tdb_valid,
+      tr: tr_valid,
+      v: v_valid,
+      met: met_valid,
+      clo: clo_valid,
+    } = check_standard_compliance_array("FAN_HEATWAVES", {
+      tdb: [tdb],
+      tr: [tr],
+      v: [v],
+      met: [met],
+      clo: [clo],
+    });
+
+    if (
+      isNaN(tdb_valid[0]) ||
+      isNaN(tr_valid[0]) ||
+      isNaN(v_valid[0]) ||
+      isNaN(met_valid[0]) ||
+      isNaN(clo_valid[0])
+    ) {
+      for (let key in joint_output) {
+        joint_output[key] = NaN;
+      }
+    }
+  }
+
+  if (joint_kwargs.round) {
+    return {
+      e_skin: round(joint_output.eSkin, 1),
+      e_rsw: round(joint_output.eRsw, 1),
+      e_max: round(joint_output.eMax, 1),
+      q_sensible: round(joint_output.qSensible, 1),
+      q_skin: round(joint_output.qSkin, 1),
+      q_res: round(joint_output.qRes, 1),
+      t_core: round(joint_output.tCore, 1),
+      t_skin: round(joint_output.tSkin, 1),
+      m_bl: round(joint_output.mBl, 1),
+      m_rsw: round(joint_output.mRsw, 1),
+      w: round(joint_output.w, 1),
+      w_max: round(joint_output.wMax, 1),
+      heat_strain_blood_flow: joint_output.heat_strain_blood_flow,
+      heat_strain_w: joint_output.heat_strain_w,
+      heat_strain_sweating: joint_output.heat_strain_sweating,
+      heat_strain: joint_output.heat_strain,
+    };
+  }
+  return joint_output;
+}
+
+/**
+ * Calculate heat strain conditions based on the result of two nodes model and max skin blood flow
+ *
+ * @param {object} two_nodes_result - the result of two nodes model
+ * @param {HeatwaveKwargs} joint_kwargs - heatwave kwargs
+ * @param {number} max_skin_blood_flow - maximum blood flow from the core to the skin, [kg/h/m2] default 80
+ * @returns {object} heat strain conditions
+ */
+function cal_heatwave(two_nodes_result, joint_kwargs, max_skin_blood_flow) {
+  let heat_strain_blood_flow,
+    heat_strain_w,
+    heat_strain_sweating,
+    heat_strain = 0;
+
+  heat_strain_blood_flow = two_nodes_result.mBl === max_skin_blood_flow ? 1 : 0;
```

Comment:
why are these variables numbers instead of booleans?
also, these comparisons are float comparisons, right? if that is the case you need to do close to rather than a comparison